lib/app_profiler.rb

```diff
@@ -162,6 +164,15 @@ def profile_sampler_enabled
       false
     end
 
+    def otel_instrumentation_enabled=(value)
+      require("opentelemetry/instrumentation/rack") if value
+
+      @otel_instrumentation_enabled = value
+    rescue Gem::LoadError => e
```

Comment:
Since we want the program to crash if this is enabled and we don't have the gem, I don't think this is necessary to rescue.

Reply:
I was preferring not to crash the program -- since we may want to enable it for all our apps via our metrics gem. Some app out in the wild may not have the otel gem for some reason and we do not want it to fail on boot.

Also, design-wise, since this is just instrumentation I think it is okay to rescue silently and disable it.

Reply:
crash it is 👍